src/SnD.Sdk/Storage/Amazon/AmazonBlobStorageService.cs

```diff
@@ -101,6 +102,24 @@ public Task<bool> RemoveBlob(string blobPath, string blobName)
             });
     }
 
+    /// <inheritdoc/>
+    [ExcludeFromCodeCoverage(Justification = "Trivial")]
+    public Uri GetBlobUri(string blobPath, string blobName, params (string, object)[] kwOptions)
+    {
+        var path = blobPath.AsAmazonS3Path();
+        var signingOptions = kwOptions.ToDictionary(opt => opt.Item1, opt => opt.Item2);
+        var duration = (double)signingOptions.GetValueOrDefault("validForSeconds", 60d);
```

Comment:
It's 60 seconds? To constant with meaningful name (`DEFAULT_VALID_SECONDS` or something like that).